dashboard/src/components/EventsContainer.js

```diff
@@ -41,7 +41,13 @@ class EventsContainer extends React.Component {
   static propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     classes: PropTypes.object.isRequired,
-    viewer: PropTypes.shape({ checkEvents: PropTypes.object }).isRequired,
+    viewer: PropTypes.shape({
+      checks: PropTypes.object,
+      entities: PropTypes.object,
+    }).isRequired,
+    environment: PropTypes.shape({
```

Comment:
So are we now filtering based on environments too? Nice.

Reply:
yup!